src/vanilla/store.ts

```diff
@@ -761,8 +763,9 @@ const deriveDevStoreRev4 = (store: Store): Store & DevStoreRev4 => {
           debugMountedAtoms.delete(atom)
         }
       }
+      atomState._ = atom.debugLabel
```

Comment:
This feels too specific. Ordinary users don't debug atomState. You want to create your own devStore for this purpose.